.github/workflows/smoke.yaml

```diff
@@ -26,6 +26,49 @@ jobs:
           path: __tests__/smoke/
           key: cache-${{ github.run_id }}-${{ github.run_attempt }}
 
+  prepare-smoke-plugins:
```

Comment:
Can we split smokes and plugin-smokes into different workflows? I feel like they should be updated separately.

Reply:
Done